lily.properties

```diff
@@ -139,3 +147,43 @@ jarfix.desc = Jarfix is a tool that can be used to fix issues where .jar files d
 snapshots.help = Informs the user about where to find snapshot builds
 snapshots.title = Where can I find snapshot builds?
 snapshots.desc = Snapshot, pre-release, and release candidate builds, when available, can be found on GitHub at https://github.com/IrisShaders/Iris/releases. You can also compile a build yourself from https://github.com/IrisShaders/Iris/tree/1.18. Per rule 5, no support will be provided with compiling Iris. Builds will also occasionally be posted in <#883067831485366304>.  Snapshot, pre-release, and release candidate builds are also sometimes made available to Patreon supporters early. \n \n **Do not ask for an ETA on when Iris will update.**
+
+source.children = iris lilybot sodium installer website
+
+source.child.iris.help = Informs the user on where to find the source code for Iris.
+source.child.iris.title = Iris on GitHub
+source.child.iris.desc = <https://github.com/IrisShaders/Iris>
+
+source.child.lilybot.help = Informs the user on where to find the source code for LilyBot.
+source.child.lilybot.title = LilyBot on GitHub
+source.child.lilybot.desc = <https://github.com/IrisShaders/LilyBot>
+
+source.child.sodium.help = Informs the user on where to find the source code for Sodium.
+source.child.sodium.title = Sodium on GitHub
+source.child.sodium.desc = <https://github.com/IrisShaders/Iris-Installer>
+
+source.child.installer.help = Informs the user on where to find the source code for the Iris installer.
+source.child.installer.title = Iris installer on GitHub
+source.child.installer.desc = <https://github.com/CaffeineMC/sodium-fabric>
+
+source.child.website.help = Informs the user on where to find the source code for the Iris website.
+source.child.website.title = Iris website on GitHub
+source.child.website.desc = <https://github.com/IrisShaders/irisshaders.github.io>
+
+discord.children = iris fabric caffeinemc
+
+discord.child.iris.help = Informs the user on where to find the Iris Discord server.
```

Comment:
This probably isn't needed, if the user is in iriscord, they probably don't need an invite to iriscord